armsrc/iso14443a.c

```diff
@@ -2359,6 +2359,7 @@ typedef struct {
   uint32_t nonce;
   uint32_t ar;
   uint32_t nr;
+  uint32_t at;
```

Comment:
Good catch. But to have two separate definitions is bad anyway. I propose that you move this type definition to include\mifare.h